classes/Product.php

```diff
@@ -7159,4 +7163,34 @@ public function softDeleteCustomizationFields($customizationIds)
 
         return $return;
     }
+
+    /**
+     * Update default supplier data
+     *
+     * @param int $idProduct
+     * @param int $idSupplier
+     * @param float $wholesalePrice
+     * @param string $supplierReference
+     *
+     * @return bool
+     */
+    public function updateDefaultSupplierData(int $idProduct, int $idSupplier, string $supplierReference, float $wholesalePrice): bool
```

Comment:
Since `$this` is not used maybe it should be a static method
Or else you should use `$this->id` instead of the `$idProduct` parameter

Reply:
Thank you're right, I added the $idProduct after for a test and keep it :sweat_smile: